pkgs/applications/misc/blender/wrapper.nix

```diff
@@ -2,17 +2,15 @@
 , lib
 , blender
 , makeWrapper
-, python3Packages
-}:
-{ name ? "wrapped"
-, packages ? []
+, extraModules ? []
```

Comment:
Nit: here in a few other places the spaces in brackets and parentheses don't seem to be used consistently? Like, sometimes it's `[ a ]`, and sometimes it's `[a]` within the same file. I think it's usually ok to mimic/preserve whatever style the file has been using

Reply:
Original file had no spaces in empty brackets, but do have spaces at each end for lists in the original file. Not changing.